samples/drivers/random/src/main.c

```diff
@@ -23,20 +23,26 @@ void main(void)
 	printf("random device is %p, name is %s\n",
 	       dev, dev->config->name);
 
+	printf("The last column is a sentinel, it always should be 0x00\n");
```

Comment:
I say we remove it, we can also reduce printing the last byte of the buffer so people don't ever see the 0x00.